Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6205,8 +5894,8 @@ public List<MediaAttachment> GetMediaAttachments(MediaAttachmentQuery query)
 
             var list = new List<MediaAttachment>();
             using (var connection = GetConnection(true))
-            using (var statement = PrepareStatement(connection, cmdText))
             {
+                using var statement = PrepareStatement(connection, cmdText);
```

Comment:
IMO it looks better to keep both `using`s outside the braces instead of mixing formats like this